docs/Manpage.md

```diff
@@ -677,6 +677,22 @@ specify *`formula`* as a required or recommended dependency for their stable bui
 * `--skip-recommended`:
   Skip all formulae that specify *`formula`* as `:recommended` type dependency.
 
+### `zap` [*`options`*] *`formula`*|*`cask`*
```

Comment:
Not this line but: should `brew cask zap` perhaps be removed from the manpage if it's deprecated (same with other commands)? That can be another PR, though.

Reply:
I think it will be removed when adding it do the `DEPRECATED_COMMANDS` list. I think we should deprecate the remaining commands in bulk in another PR.

Reply:
👍🏻 Cool. If it doesn't: it'd be good to add that as part of that PR.